public/docs/ts/latest/cookbook/angular-in-dotnet-core.jade

```diff
@@ -0,0 +1,290 @@
+include ../_util-fns
+
+:marked
+  This cookbook describes how to build an Angular app inside the MVC architecture in a .NET Core Web application. It does not explain how Angular or .NET Core works. However it will provide all the code snippets required. Therefore you will be able to follow through this cookbook even if you are currently not familiar with Angular or .NET Core.
+
```

Comment:
Perhaps it may be helpful to have a short section here that identifies who the audience is